Godeps/Godeps.json

```diff
@@ -370,6 +370,11 @@
 			"Rev": "63ce630574a5ec05ecd8e8de5cea16332a5a684d"
 		},
 		{
+			"ImportPath": "github.com/aws/aws-sdk-go/service/kms",
+			"Comment": "v1.6.10",
+			"Rev": "63ce630574a5ec05ecd8e8de5cea16332a5a684d"
```

Comment:
is upstream on this level too?

Reply:
yes, current kubernetes master is on v1.6.10